server/clients/manageRecallsApi/updateRecall.pact.test.ts

```diff
@@ -41,6 +41,45 @@ pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, prov
       expect(actual).toEqual(updateRecallResponseJson)
     })
 
+    test('can successfully add last release date to a recall', async () => {
+      const lastReleaseDateTime = '2020-08-03T00:00:00.000Z'
+      await provider.addInteraction({
+        state: 'a recall exists and can be updated with last release date',
+        ...updateRecallRequest('an update recall request', recallId, { lastReleaseDateTime }, accessToken),
+        willRespondWith: updateRecallResponse(Matchers.like(updateRecallResponseJson), 200),
+      })
+
+      const actual = await updateRecall(recallId, { lastReleaseDateTime }, accessToken)
+
+      expect(actual).toEqual(updateRecallResponseJson)
+    })
+
+    test('can successfully add last releasing prison to a recall', async () => {
+      const lastReleasePrison = 'Belmarsh'
+      await provider.addInteraction({
+        state: 'a recall exists and can be updated with last releasing prison',
+        ...updateRecallRequest('an update recall request', recallId, { lastReleasePrison }, accessToken),
+        willRespondWith: updateRecallResponse(Matchers.like(updateRecallResponseJson), 200),
+      })
+
+      const actual = await updateRecall(recallId, { lastReleasePrison }, accessToken)
+
+      expect(actual).toEqual(updateRecallResponseJson)
+    })
+
+    test('can successfully add local police service to a recall', async () => {
+      const localPoliceService = 'Brentwood, Essex'
+      await provider.addInteraction({
+        state: 'a recall exists and can be updated with local police service',
+        ...updateRecallRequest('an update recall request', recallId, { localPoliceService }, accessToken),
+        willRespondWith: updateRecallResponse(Matchers.like(updateRecallResponseJson), 200),
+      })
+
+      const actual = await updateRecall(recallId, { localPoliceService }, accessToken)
+
+      expect(actual).toEqual(updateRecallResponseJson)
+    })
+
```

Comment:
Do we want to add a test for each property? Or should we test them all at once? We are having similar discussion on the backend...

Reply:
I think separate tests make it clear that you can update any independently, although obvs more boilerplate
I don't feel really strongly either way